packages/seal/src/Search/Result.php

```diff
@@ -32,4 +32,11 @@ public function total(): int
     {
         return $this->total;
     }
+
+    public static function empty(): static
```

Comment:
as static function can also be called unstatic way I would prefer `createEmpty`, what do you think?